R/binary_classification.R

```diff
@@ -76,3 +76,44 @@ ll <- function(actual, predicted) {
 logLoss <- function(actual, predicted) {
     return(mean(ll(actual, predicted)))
 }
+
+
+
+#' Precision
+#' 
+#' \code{precision} computes proportion of predicted 1's that are actual 1's
+#' @export
+#' @examples 
+#' actual <- c(1, 1, 1, 0, 0, 0)
+#' predicted <- c(1, 1, 1, 1, 1, 1)
+#' precision(actual, predicted)
+precision <- function(actual, predicted) {
+  return(mean(actual[predicted == 1]))
+}
+
+#' Recall
+#' 
+#' \code{recall} computes proportion of actual 1's that are predicted 1's
+#' @export
+#' @examples 
+#' actual <- c(1, 1, 1, 0, 0, 0)
+#' predicted <- c(1, 0, 1, 1, 1, 1)
+#' recall(actual, predicted)
```

Comment:
Can you add all of the same documentation improvements that were listed for `precision`?